.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: objective-c
-osx_image: xcode10.1
+osx_image: xcode11.1
```

Comment:
I should update CI completely to support the new Xcode. Thanks for taking care of this though!